Review governance pages (#367)
trilitech/tezos-developer-docs: docs/architecture/governance/improvement-process.md

---
As Tezos is a large decentralized project that is constantly evolving, its members need to have a mechanism for proposing improvements to the ecosystem.

> A Tezos Improvement Proposal (or TZIP, pronounced "tee-zip") is a document that offers ways to improve Tezos via new features, tools, or standards (e.g. smart contract interface specifications).

Reviewer: Don't use Latinate abbreviations like "e.g."

Reviewer: Style guides generally recommend against Latinate abbreviations like e.g. and i.e but it's not a big deal.

Author: Of course, the importance is limited, but I'm asking this because I couldn't find this kind of recommendation in the Chicago or AP guides.
Thanks!

> Specifically, a TZIP is a design document proposed to the Tezos community, describing a feature for Tezos or for the related processes, tools, or artefacts (e.g., smart contracts).

Reviewer: ```suggestion
Specifically, a TZIP is a design document proposed to the Tezos community, describing a feature for Tezos or for the related processes, tools, or artifacts (such as smart contracts).
```

TZIPs address a different need when compared to the formal online governance process for protocol amendments.
Typically, TZIPs do not impact the Tezos protocol at all. They rather concern other parts of the Tezos ecosystem.

> In theory, a TZIP could support the formal online governance process by defining a specification including some required protocol change, that can later be implemented in a protocol amendment. In such an eventuality, the TZIP process would complement (but remain subsidiary to) Tezos' formal on-chain governance process.

Reviewer: Is this hypothetical important enough to include here?

Reviewer: I think the statement about this potential use of TZIP is confusing because it says how TZIP could be part of the formal governance process, but the previous paragraph says that they are not part of the formal process. Maybe a better example would be that they create token standards that are not required but something that the community agrees on.
